tc/core/compiler.h

```diff
@@ -117,6 +119,12 @@ std::unique_ptr<typename Backend::ExecutorType> compile(
 std::vector<TensorInfo> inferOutputTensorInfo(
     lang::TreeRef tcDefinition,
     const std::vector<const DLConstTensor*> inputs);
+
+void helpThrowInvalidStride(
```

Comment:
this should definitely not be exposed to the high-level API (at least not with this name and signature)